src/ResourceManager/Compute/Commands.Compute/Strategies/Compute/VirtualMachineStrategy.cs

```diff
@@ -31,7 +31,7 @@ static class VirtualMachineStrategy
                     p.ResourceGroupName, p.Name, null, p.CancellationToken),
                 createOrUpdateAsync: (o, p) => o.CreateOrUpdateAsync(
                     p.ResourceGroupName, p.Name, p.Model, p.CancellationToken),
-                createTime: c => c.OsProfile.WindowsConfiguration != null ? 240 : 120);
+                createTime: c => c.OsProfile == null ? 240 : c.OsProfile.WindowsConfiguration != null ? 240 : 120);
```

Comment:
Do not use nested ? :  
Combine two conditions or use if-then-else.

Reply:
This is a lamda expression, so I just want to fit it into one line...

Reply:
Anyway, I will try not to use nested here.